test/unit/mock/fake_runtime_service.py

```diff
@@ -52,11 +52,11 @@ def __init__(self, *args, **kwargs):
         ):
             super().__init__(*args, **kwargs)
 
-    def _authenticate_legacy_account(self, client_params: ClientParameters):
+    def _authenticate_legacy_account(self, client_params: ClientParameters) -> Any:
```

Comment:
Does `-> FakeAuthClient` not work?

Reply:
```
Return type "FakeAuthClient" of "_authenticate_legacy_account" incompatible with return type "AuthClient" in supertype "IBMRuntimeService"
```

There is a mypy override error if `FakeAuthClient` is used as a return type so we can have `Any` as a return type or do something like ```-> 'FakeAuthClient' # type: ignore```